src/openvic-simulation/map/ProvinceInstance.hpp

```diff
@@ -159,9 +160,11 @@ namespace OpenVic {
 		bool setup(BuildingTypeManager const& building_type_manager);
 		bool apply_history_to_province(ProvinceHistoryEntry const& entry, CountryInstanceManager& country_manager);
 
-		void initialise_for_new_game(ModifierEffectCache const& modifier_effect_cache);
+		void initialise_rgo(ModifierEffectCache const& modifier_effect_cache);
 
 		void setup_pop_test_values(IssueManager const& issue_manager);
+		State* get_mutable_state();
 		plf::colony<Pop>& get_mutable_pops();
+		IndexedMap<PopType, std::vector<Pop*>>& get_mutable_pops_cache_by_type();
```

Comment:
This isn't needed, we can access the mutable `Pop*`s via the const getter and we shouldn't be modifying the cache itself anywhere outside of the `ProvinceInstance`.